pysoa/common/logging.py

```diff
@@ -81,6 +81,7 @@ class RecursivelyCensoredDictWrapper(object):
         'auth', 'token', 'auth_token', 'auth-token', 'authToken',
         'authorization', 'authorization_token', 'authorization-token', 'authorizationToken',
         'authentication', 'authentication_token', 'authentication-token', 'authenticationToken',
+        'invitation_token',
```

Comment:
no alternative spellings (`invitation`, `invitation-token`, `invitationToken`)?

Reply:
Not foreseeing the variations ever being used in the context of pysoa.